tests/bearlib/languages/documentation/DocumentationCommentTest.py

```diff
@@ -61,14 +69,6 @@ def check_docstring(self, docstring, expected=[]):
         parsed_metadata = doc_comment.parse()
         self.assertEqual(parsed_metadata, expected)
 
-    def load_testdata(self, filename):
```

Comment:
[here](https://github.com/coala-analyzer/coala/pull/2390/commits/e2cf672763ddf01e9a00e9199e3ad65c3049065e#diff-603a58b800b18edbad56a01690a21e05R124) 124 and [here](https://github.com/coala-analyzer/coala/pull/2390/commits/e2cf672763ddf01e9a00e9199e3ad65c3049065e#diff-603a58b800b18edbad56a01690a21e05R95) 95